src/test/java/juja/sqlcmd/DatabaseManagerTest.java

```diff
@@ -10,65 +10,80 @@
 import java.sql.DriverManager;
 import java.sql.SQLException;
 import java.sql.Statement;
-import java.util.Date;
 
-import static junit.framework.TestCase.assertFalse;
 import static org.junit.Assert.assertArrayEquals;
 import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
 import static org.junit.Assert.assertTrue;
 
 public class DatabaseManagerTest {
-    private static final String DB_NAME = "sqlcmd";
-    private static final String USER = "sqlcmd";
-    private static final String PASSWORD = "sqlcmd";
+    private static final String NAME_DB_TEST = "sqlcmd_test";
+    private static final String LOGIN_FOR_TEST = "sqlcmd";
+    private static final String PASSWORD_FOR_TEST = "sqlcmd";
+    private static final String LOGIN_FOR_SU = "postgres";
+    private static final String PASSWORD_FOR_SU = "postgres";
     private static final String TEST_DB_CONNECTION_URL = "jdbc:postgresql://localhost:5432/";
     private static final String TABLE_NAME = "table_name";
 
     private static Connection connection;
-    private static String dbName;
 
     private DatabaseManager databaseManager;
 
     @BeforeClass
     public static void createConnection() throws SQLException, ClassNotFoundException {
         Class.forName("org.postgresql.Driver");
-        connection = DriverManager.getConnection(TEST_DB_CONNECTION_URL, USER, PASSWORD);
-        dbName = new Date().toString();
+        connection = DriverManager.getConnection(TEST_DB_CONNECTION_URL, LOGIN_FOR_SU, PASSWORD_FOR_SU);
         try (Statement statement = connection.createStatement()) {
```

Comment:
Why don't you use your own "executeQuery" method?

Reply:
Well it's not possible while method executeQuery() is not static... And why it's not static - that is next question. As for me this util method should be static, why to create it per each instance?

Reply:
Thanks for the comments, I'll think about it.
Perhaps in the next PR I will do service methods, such as executeQuery(), static